### ISSUE TITLE
Investigate CMPS11

### ISSUE THREAD

**Issue by Danyc0** [abersailbot/kitty-hardware#15]
The CMPS10s are really useless, but there is a new version, the CMPS11, which should probably be tested as a much cheaper alternative to the HMC6343. And when I say tested, I mean super heavily tested, because the testing we did on the CMPS10's in 2014 was not good enough, because it didn't throw up the issues until Sailbot2014, at which point we had no replacement.

**Comment by naturesyouth** [comment]
surely the best way to test this would be to have all three fixed to the same boat and then comparing the data they spit out when actually testing the boats.

**Comment by kragniz** [comment]
Building a board (literal board, not a pcb) with all three attached would also do the same thing and doesn't require physically attaching them into a boat.

**Comment by Danyc0** [comment]
We did that test, on a board, back in 2014, between the CMPS10 and the HMC6343, it put the CMPS10 as "good enough", however later experience tells us that was not the case!

**Comment by naturesyouth** [comment]
true, but adding them into dewi and then running her on some actual sailing tasks would give real world data

**Comment by Danyc0** [comment]
Real world data would not have detected the CMPS10 issue. It needs to be long term testing, running for days, a week maybe. As well as comparisons with the HMC6343.
